### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by sgomez**
Add bootstrap pagination menu in manage and admin views. PR #4 must be merged first.

### CODE REVIEW

File: src/Repository/ShortUrlRepository.php [hunk 1]
```diff
@@ -17,20 +19,38 @@
  */
 final class ShortUrlRepository extends ServiceEntityRepository
 {
+    public const NUM_ITEMS_PER_PAGE = 10;
```
[the remark this answers is not in the document]
Comment (sgomez) [reply src/Repository/ShortUrlRepository.php]: I added a parameter to configure pagination (100 by default). But this kind of functionality (search, pagination) should be better to be done with javascript (xhr request).

[the remark this answers is not in the document]
Comment (sgomez) [reply src/Repository/ShortUrlRepository.php]: Added a APP_SHORTLINK_PAGINATION parameter to configure links per page.

**Comment by sgomez** [comment]
Last changes:

* Rebased the branch to master and added a search input box.
* New strings in d0823a6 that should be translated.

**Review by thijskh — COMMENTED** [review]
Looks good!

Can we however also search on the short link? I have a short link that I want to update. Maybe I want to search by that link's name instead of it's target?

For the admin interface, would we be able to also filter on the creator (persistent id)?

**Comment by sgomez** [comment]
@thijskh added in last commit. But the search field should be selected with a select input.

**Review by thijskh — COMMENTED** [review]
Why is there a need for the user to select the field to search for? The three fields to search on will not have huge overlaps in hits. So if I type a short link I will likely not get many unintended hits, same for long url, same for persistent ID.

Also I think it would be great if you could just paste a short link into the search form and get the relevant hit ("https://edulnk.org/bvcds" would match, or edulnk.org/bvcds, not just bvcds)

**Comment by sgomez** [comment]
This platform could have millions of rows and we need to use LIKE search for partials (not indexable). Without the select field the RDBM should do this two/three times for request (one for each column). 

Anyway, if the main issue is resolved, improvements on filtering can be done in another PR.

**Comment by thijskh** [comment]
I'm stipulating those because the things I describe _are_ possible now.

At least also allowing to search for ((https://)edulnk.org/)<shortcode> aswell should not incur a database penalty.

**Comment by thijskh** [comment]
About the database hit; for the end user UI you'd use an exact match on the indexable field _owner_ first (which you already need to do), then the partial matchers are only applied to that subset, usually a handful or in the order of dozens, not millions of rows. I can see the drawback for the admin UI, there simplicity of use is less of an issue for me.

**Comment by sgomez** [comment]
I have not seen it from that point of view, if owner and shorlUrl are a full match search there shouldn't be much penalty.

Updated in last commit.

**Review by thijskh — APPROVED** [review]

**Pull request merged**